client/src/Components/Robot.js

```diff
@@ -39,12 +63,12 @@ class Robot extends Component {
   }
 
   render() {
-    // TODO: change play button depending on whether melody is currently playing.
     return (
       <div>
         <div className="robot-toolbar">
           <MDBIcon
```

Comment:
Rather than changing whether the play button is filled or not, can we change the play button to a pause button?